.eslintrc.json

```diff
@@ -13,28 +13,26 @@
 		"plugin:promise/recommended",
 		"plugin:jest/recommended",
 		"plugin:jest/style",
-		"prettier",
-		"prettier/@typescript-eslint"
```

Comment:
Why did you remove this?

Reply:
Upgraded `esling-config-prettier`, turns out they have removed the second config because it's built in to the first.